src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -1262,6 +1262,205 @@ TEST_F(SetGlobalPropertiesRequestTest,
   command->Run();
 }
 
+TEST_F(SetGlobalPropertiesRequestTest,
+       Run_InvalidCustomizableKeys_INVALID_DATA) {
+  MessageSharedPtr msg = CreateMsgParams();
+  SmartObject customizable_keys(smart_objects::SmartType_Array);
+  customizable_keys[0] = "%";
+  customizable_keys[1] = "\\n";
+  customizable_keys[2] = " ";
+  (*msg)[am::strings::msg_params][am::strings::keyboard_properties]
+        [am::hmi_request::customize_keys] = customizable_keys;
```

Comment:
@ychernysheva good candidate for moving out to a separate function in the `SetGlobalPropertiesRequestTest`

Reply:
Done in f10bc21